cpp/ql/lib/semmle/code/cpp/ir/dataflow/internal/DataFlowUtil.qll

```diff
@@ -100,7 +100,7 @@ class Node extends TIRDataFlowNode {
   Declaration getEnclosingCallable() { none() } // overridden in subclasses
 
   /** Gets the function to which this node belongs, if any. */
-  Function getFunction() { none() } // overridden in subclasses
+  Declaration getFunction() { none() } // overridden in subclasses
```

Comment:
Because we're no longer talking (exclusively) about functions - we can now have code in global variable initializers that shows up in the dataflow graph, and there's places that assume `getFunction` has a result for those nodes.

Reply:
Thanks for the explanation. I was aware of the related changes in https://github.com/github/codeql/pull/8515. It does make me wonder whether we should spend some time renaming predicates to better reflect what they're doing (doesn't have to be in this PR).

Reply:
I also see that in some places `func` was renamed to `decl` and in other places it wasn't. Is that on purpose?